MvvmCross/Platforms/Android/Core/MvxAndroidSetup.cs

```diff
@@ -231,8 +231,6 @@ protected virtual IEnumerable<Assembly> ValueConverterAssemblies
         protected virtual IDictionary<string, string> ViewNamespaceAbbreviations => new Dictionary<string, string>
         {
             { "Mvx", "mvvmcross.platforms.android.views"
-            },
-            { "Mvx", "mvvmcross.platforms.android.binding.views"
             }
         };
```

Comment:
mvvmcross.platforms.android.binding.views is the original and should be the one to keep. Otherwise breaking changes occurs.